pages/How-to/guide/adding-issue-template.md

```diff
@@ -1,3 +1,66 @@
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/74750414/169188784-680014eb-ae5a-4062-b9c0-40276f28a261.png" />
-</div>
+# How to create issue templates ?
+An issue template is a template with prepopulated information about how an issue can be mentioned.
+
+## Step by step process to create an issue template :
+
+### Step 1 :
+
+You should have a repository. If you don't, go on <u><a href = "https://oswp.study/#/pages/How-to/guide/repo-private-public">this</a></u> page to know how to create a repository.
```

Comment:
```suggestion
You should have a repository.
```

Reply:
That Docs will be for making the repo private/public not for creating a repo. So it will not be suitable.